src/System.Memory/tests/BuffersExtensions/BuffersExtensionsTests.cs

```diff
@@ -29,6 +29,75 @@ public void WritingToMultiSegmentBuffer()
             Assert.Equal("Hello World!", bufferWriter.ToString());
         }
 
+        [Fact]
+        public void WritingEmptyBufferToSingleSegmentEmptyBufferWriterDoesNothing()
+        {
+            IBufferWriter<byte> bufferWriter = new MultiSegmentArrayBufferWriter<byte>(
+                new byte[][] { Array.Empty<byte>() }
+            );
+
+            bufferWriter.Write(Array.Empty<byte>()); // Span<byte>.Empty.CopyTo(Span<byte>.Empty);
```

Comment:
Can you add a text like "equivalent to:" within the comment so that it doesn't look like accidentally commented out code.

```suggestion
            bufferWriter.Write(Array.Empty<byte>()); // This is equivalent to: Span<byte>.Empty.CopyTo(Span<byte>.Empty)
```